tests/py/translations_test.py

```diff
@@ -1,13 +1,19 @@
+from unittest.mock import Mock
+
 import pytest
 
-from qiskit_sphinx_theme.translations import get_language_label, get_translation_url
+from qiskit_sphinx_theme.translations import (
+    extend_html_context,
+    get_language_label,
+    get_translation_url,
+)
 
 
 @pytest.mark.parametrize(
     "docs_url_prefix,page,expected",
     [
-        ("", "index", "/index.html"),
-        ("", "subdir/my_page", "/subdir/my_page.html"),
```

Comment:
We now error if you set `""` because it won't work for any project we have in the wild: they _all_ have a prefix.

The only one I was confused by is Terra. They set this value to `documentation`, but in a weird way in some scripts that took me a while to find. So, at first, I incorrectly thought Terra was setting the value to `""`.